packages/nextjs/src/utils/serverIntegrations.ts

```diff
@@ -0,0 +1,60 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+const SOURCEMAP_FILENAME_REGEX = /^.*\/.next\//;
+
+/** Default RewriteFrames integration to match filenames in Sentry. */
+export const REWRITE_FRAMES_INTEGRATION = new RewriteFrames({
+  iteratee: frame => {
+    try {
+      if (frame.filename) {
+        frame.filename = frame.filename.replace(SOURCEMAP_FILENAME_REGEX, 'app:///_next/');
+      }
```

Comment:
```suggestion
      frame.filename = frame.filename.replace(SOURCEMAP_FILENAME_REGEX, 'app:///_next/');
```

Because we have the `try...catch`, we don't really need to add the `if`.